<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QWidget" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">gpusim-fe - &lt;Project file name&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,1">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QFrame" name="frmToolbar">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="frmToolbarLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnNew">
        <property name="text">
         <string>New Experiment</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_new.png</normaloff>:/buttons/btn_new.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnOpen">
        <property name="text">
         <string>Open Experiment</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_open.png</normaloff>:/buttons/btn_open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSave">
        <property name="text">
         <string>Save Experiment</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_save.png</normaloff>:/buttons/btn_save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSaveAs">
        <property name="text">
         <string>Save Experiment As...</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_saveAs.png</normaloff>:/buttons/btn_saveAs.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="toolbarSeparator_2">
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnApply">
        <property name="text">
         <string>Apply changes</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_apply.png</normaloff>:/buttons/btn_apply.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="toolbarSeparator">
        <property name="frameShape">
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRun">
        <property name="text">
         <string>Run Experiment</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_runModeling.png</normaloff>:/buttons/btn_runModeling.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnViewPlots">
        <property name="text">
         <string>View Experiment Plots</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_viewPlots.png</normaloff>:/buttons/btn_viewPlots.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnAbout">
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset resource="Resources/resources.qrc">
          <normaloff>:/buttons/btn_about.png</normaloff>:/buttons/btn_about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonIconOnly</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmExperimentEditor">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="frmExperimentEditorLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnNew</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>newExperiment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpen</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>openExperiment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>saveExperiment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>saveExperimentAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnApply</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRun</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>runExperiment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnViewPlots</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>viewExperimentPlots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAbout</sender>
   <signal>clicked()</signal>
   <receiver>CMainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newExperiment()</slot>
  <slot>openExperiment()</slot>
  <slot>saveExperiment()</slot>
  <slot>saveExperimentAs()</slot>
  <slot>apply()</slot>
  <slot>runExperiment()</slot>
  <slot>viewExperimentPlots()</slot>
  <slot>about()</slot>
 </slots>
</ui>
